src/extension.ts

```diff
@@ -18,7 +19,7 @@ export function activate(context: vscode.ExtensionContext) {
     });
 
     let runterminal = vscode.commands.registerCommand('vsc-extension-ansible.ansible-terminal', () => {
```

Comment:
we should change the command to "run ansible docker in terminal". this naming is also related to what's usage scenario of terminal..

Reply:
well, i think "run ansible terminal" is correct, it will run user's default configuration.
perhaps we should add "configure ansible terminal" so user can choose options like "docker / local / cloudshell", and then based on that selection we store appropriate configuration, so "run ansible terminal" knows what's default.

Reply:
or "configure ansible" and that should be done as a separate task